Structure_Engine/Query/Geometry.cs

```diff
@@ -23,7 +23,7 @@
 using BH.oM.Geometry;
 using BH.oM.Structure.Elements;
 using BH.oM.Structure.SectionProperties;
-using BH.Engine.Common;
+using BH.Engine.Spatial;
```

Comment:
This is not even used actually, so might as well get rid of it isntead of replacing